src/main/java/taxiapp/service/ManufacturerService.java

```diff
@@ -0,0 +1,18 @@
+package taxiapp.service;
+
+import java.util.List;
+import java.util.Optional;
+import taxiapp.model.Manufacturer;
+
+public interface ManufacturerService {
+
+    Manufacturer create(Manufacturer manufacturer);
+
+    Optional<Manufacturer> get(Long manufacturerId);
```

Comment:
```suggestion
   Manufacturer get(Long manufacturerId);
```
on the service layer, we return an object not its optional